handbook/business-operations/README.md

```diff
@@ -858,6 +858,35 @@ To make a request of the business operations department, [create an issue using
 
 > If you're not sure that your request can wait that long, then please ask for urgent help in our group Slack channel: `#g-business-operations`.  Only use this approach or at-mention contributors in business operations directly in urgent situations.  Otherwise, create an issue.
 
+## Contract review
+
+> If a document is ready for signature and does not need to be reviewed or negotiated, you can skip the review process and use the signature process documented further down in this handbook section.
+
+**To get a contract reviewed:**
+
+Upload the agreement to [Google Drive](https://drive.google.com/drive/folders/1G1JTpFxhKZZzmn2L2RppohCX5Bv_CQ9c).
+
+Complete the [contract review issue template in GitHub](https://fleetdm.com/handbook/business-operations#intake), being sure to include the link to the document you uploaded and using the Calendly link in the issue template to schedule time to discuss the agreement with Nathan Holliday.
+
+Follow up comments should be made in the GitHub issue and in the document itself so it is all in the same place.
+
+The SLA for contract review is **2 business days**.
+
+Once the review is complete, the issue will be closed.
+
+If an agreement requires an additional review during the negotiation process, the requestor will need to follow these steps again. Uploading the new draft and creating a new issue in GitHub. 
+
+When no further review or action is required for an agreement and the document is ready to be signed, the requestor is then responsible for routing the document for signature.
+
+
+## Contract signature
+
+When a contract is ready for signature, the requestor logins into DocuSign using hello@ from the 1Password vault and routes the agreement to Mike for signature. 
```

Comment:
Looks like Mike's requested change didn't make it in here ("to Mike for signature" ➡️ "to the CEO for signature")